src/story/element/mod.rs

```diff
@@ -118,3 +104,63 @@ pub fn denormalize_element(
         }
     }
 }
+
+// TODO: ***technically*** this is only a view concern but keeping it here for now.
```

Comment:
This is a view concern because ElementExposure is only used for rendering. I can probably get this working in the view layer with a little effort, but it sounded like we might delete this code soon too. It doesn't seem like a huge deal to track it on the model right now. The biggest issue is that fast-forward takes a perf hit calculating element exposure when it's not rendering.